_posts/JayYang95/2016-04-28-SecondMeetup.md

```diff
@@ -0,0 +1,14 @@
+---
+layout: post
+author: JayYang95
+title: "Second Meet-Up"
+---
+
+For my second meet-up, I attended another TriPython meeting in the same place, the Europa center, on Thursday, April 28. This meet-up was quite different from the first one. Instead of everyone scattered around tables openly discussing their projects and such, we were all gathered around a speaker and intently listening to him. The speaker this time was Rex Dwyer, a professor of Computer Science from NCSU. When I came in, I thought I would have no idea/knowledge of what we would be listening to. Surprisingly, professor Dwyer spoke about something that was actually relevant to our class, which is regexes. For most of the talk, he showed us the features of a new regex module called "regex" vs the module we use in class which is "re". He began by saying that he had tried some kind of regex on the re module that didn't work, so he reported it as a bug. It turned out that the regex he used simply did not work for the "re" module but now works in the new "regex" module. I was not quite sure exactly what kind of problem he ran into, but it had to do with the split function. Unfortunately, towards the end of the talk, the regex examples used were so long and complicated that I wasn't able to follow what they did since they used regex syntax I wasn't familiar with. Here is the link to their module:
+
+https://pypi.python.org/pypi/regex
+
+I think what's really cool is how related this was to our class. Perhaps in the future, this class may even be taught using the new regex module for exercises instead of re. 
```

Comment:
QUite possible- it's a good example of how the stdlib modules like `re` can be powerful but sometimes people feel like writing their own to replace them